src/main/java/app/oengus/entity/model/EmailVerification.java

```diff
@@ -0,0 +1,33 @@
+package app.oengus.entity.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
```

Comment:
Star imports are considered code smell, tip is to set your IDE to only do star imports at a very high number (99 or higher).